source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -3761,77 +3809,18 @@ export class AdaptiveCard extends ContainerWithActions {
     updateLayout(processChildren: boolean = true) {
         super.updateLayout(processChildren);
 
-        if (AdaptiveCard.useAdvancedCardBottomTruncation) {
-            this.handleBottomOverflows();
+        if (AdaptiveCard.useAdvancedCardBottomTruncation && this.isRendered()) {
+            var card = this.renderedElement;
+            var padding = this.hostConfig.getEffectivePadding(Enums.Padding.Default);
+            var boundary = card.offsetTop + card.offsetHeight - padding + 1;
+
+            this.handleBottomOverflow(boundary);
```

Comment:
Why not call this.truncateOverflow instead?

Reply:
The thought was that calling `truncateOverflow` would be a bit of a misnomer because it also handles the case where content is expanded due to more room becoming available.

Containers are a bit different in that the code in `handleBottomOverflow` is optimized so that we don't actually need to call `resetOverflow` for them -- `handleBottomOverflow` expands the content if needed. This lets us avoid triggering multiple element visibility changed events in succession, where an element might become visible due to resetting overflow, but then immediately be hidden again.

I'll look into moving some stuff around to see if we can make this clearer.